pyproject.toml

```diff
@@ -49,7 +49,7 @@ click = ">=7.1.2,<9.0"  # NOTE: COLAB has 7.1.2 and has problems updating
 rich = "^10.15.2"
 shellingham = "^1.4.0"
 scikit-learn = ">=0.24.0"
-cloudpickle = "^1.6.0"
```

Comment:
Does this mean we'll have to pin the version of cloudpickle inside of the async/sync inference endpoints to 2.0.0? Not entirely sure actually

Reply:
Nope!

Technically to my understanding and exploring the cloudpickle source code, you can use even simple pickle to deserialize